script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -129,11 +129,11 @@ This not only damages the name of the family but also lowers the quality of the
 Those who cannot take on that responsibility are not qualified to be parents, and children who cannot live up to that responsibility have no reason to be part of a family.
 Tohno Makihisa told this to me and Akiha many times. It was as if he was persuading himself, over and over again.//　遠野槙久はこんな話を何度も俺と秋葉に語った。まるで自分自身に言い聞かせるように、何度も何度も。
 <Page38>
-"That was one of his favorite phrases, wasn't it.//「<槙久|おやじ>の口癖だったな。口癖 is like "favorite phrase"/"pet saying"/etc.
+"That was one of his favorite phrases, wasn't it?//「<槙久|おやじ>の口癖だったな。口癖 is like "favorite phrase"/"pet saying"/etc.
 So the reason you called me back is simply..."
 It wasn't consideration for her brother, who was disowned by their father.//　<偏屈|へんくつ>な父に勘当された兄の心情をおもんぱかっての事ではなく、心情 is missing here but it reads ok.
 <Page39>
-"It's because regardless of how we got here, you are the eldest son of the Tohno family.//「どのような経緯であれ、貴方が遠野家の長男だからです。
+"It is because regardless of how you got here, you are the eldest son of the Tohno family.//「どのような経緯であれ、貴方が遠野家の長男だからです。
```

Comment:
I think the first part is more like "Regardless of the details", not Shiki arriving, hence the `we`

Reply:
I'm also not sure why this contraction was dropped. revert imo